### PR TITLE
Henry/ Added ApplicationPage component as well as its child components.

### PR DESCRIPTION
**Pull request by henryaudi**

### CODE REVIEW

File: src/pages/apply/ApplicationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+import CancelIcon from '@mui/icons-material/Cancel';
+import '@fontsource/roboto/300.css';
+import '@fontsource/roboto/400.css';
+import '@fontsource/roboto/500.css';
+import '@fontsource/roboto/700.css';
+import FileUpload from '../../components/FileUpload';
+import BottomPanel from '../../components/BottomPanel';
+
+// Static variable field
+const courses = [
```
Comment (LonesomeTown) [inline src/pages/apply/ApplicationPage.tsx:27]: Add TODO here, we gonna use data from the backend in the future.

File: src/pages/apply/ApplicationPage.tsx [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+                  label="Last Name"
+                  name="lastName"
+                  autoComplete="family-name"
+                  onChange={(e) => setLastName(e.target.value)}
+                  value={lastName}
+                  variant="standard"
+                />
+              </Grid>
+
+              <Grid item xs={12} sx={{ mt: 2 }}>
```
Comment (LonesomeTown) [inline src/pages/apply/ApplicationPage.tsx:187]: We don't show course id on the UI, it's only passed as one of the parameters and stored in the database.

File: src/pages/login/SignUpPage.tsx [hunk 3]
```diff
@@ -17,6 +17,7 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
 import Copyright from '../../components/Copyright';
 import { AxiosError } from 'axios';
+import FileUpload from '../../components/FileUpload';
```
Comment (LonesomeTown) [inline src/pages/login/SignUpPage.tsx:20]: Do we need it here?

File: src/pages/apply/ApplicationPage.tsx (hunk 1 above)
Comment (LonesomeTown) [inline src/pages/apply/ApplicationPage.tsx:27]: Add TODO here, we gonna use data from the backend in the future.

File: src/pages/apply/ApplicationPage.tsx [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+                <Typography component="h2" variant="h6" justifyContent="center">
+                  Course ID
```
Comment (LonesomeTown) [inline src/pages/apply/ApplicationPage.tsx:189]: We don't show course id on the UI, it's only passed as one of the parameters and stored in the database.

File: src/pages/login/SignUpPage.tsx (hunk 3 above)
Comment (henryaudi) [reply src/pages/login/SignUpPage.tsx:20]: No, Idk why it was there..

**Review by LonesomeTown — APPROVED** [review]
LGTM!

**Review by LonesomeTown — APPROVED** [review]
LGTM!

**Pull request merged**